Monika After Story/game/script-farewells.rpy

```diff
@@ -1562,3 +1562,35 @@ label bye_prompt_restart:
     $ persistent._mas_greeting_type_timeout = datetime.timedelta(minutes=20)
     $ persistent._mas_greeting_type = store.mas_greetings.TYPE_RESTART
     return 'quit'
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_workout",
+            unlocked=True,
+            prompt="I'm going to work out.",
+            pool=True
+        ),
+        code="BYE"
+    )
+
+label bye_workout:
+    if mas_isMoniNormal(higher=True):
+        m 1eua "Oh, going to the gym [player]? Or perhaps for a jog?"
+        if persistent._mas_pm_love_yourself is True:
+            m 3tub "I can only imagine how sexy you look when you've worked up a sweat...{fast}{nw}"
```

Comment:
Pretty sure you meant this to be a `{cps=*2}` instead of just a `{fast}{nw}`, since you'd basically never see this at all.

That said, I also don't like this line. At all. It's out of character, and I think it'll make people uncomfortable (she also rejects this as a name/nickname to call you)

Even if this is for `persistent._mas_pm_love_yourself` being `True`, I can assure you that some people might not like being called sexy or something out of the blue like that.

Reply:
Agreed, I'll rewrite this line.

Reply:
How about this?

m 3tub "{cps=*2}I can only imagine how good you look when you've worked up a sweat...{nw}"